tests/loopbacked_tests.rs

```diff
@@ -85,24 +109,15 @@ fn test_with_spec<F>(count: u8, test: F) -> ()
 {
     init_logger();
     let tmpdir = TempDir::new("stratis").unwrap();
-    let loop_devices: Vec<LoopDevice> = get_devices(count, &tmpdir);
+    let loop_devices: Vec<LoopTestDev> = get_devices(count, &tmpdir);
     let device_paths: Vec<PathBuf> = loop_devices
         .iter()
-        .map(|x| x.get_path().unwrap())
-        .collect();
+        .map(|x| x.get_path())
+        .collect::<Vec<PathBuf>>();
```

Comment:
This type annotation ought not to be necessary, since the one above should do the trick.